app/assets/javascripts/save_functions.js

```diff
@@ -19,8 +19,11 @@ window.send_correction = function(
     current_value,
     correct_answer,
     quota,
-    round_number
+    round_number,
+    phase
 ) {
+    phase_int = phase === 'phase_one' ? 1 : 2
```

Comment:
Is there anyway we could make this less "magic numbery"?  Is phase_int, the backend phase ID?